package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "name": "@storybook/root",
-  "version": "1.0.0",
+  "version": "1.0.0-beta.1",
```

Comment:
why?

Reply:
Um... this was the wrong file. I meant to do this in the preset. Code review win.